### PR TITLE
yeeeStudentNumber

### PR DESCRIPTION
**Pull request by TWBestATM**

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+    printf("1023027");
```
Comment (jrjang2) [inline whoami.c]: 1. Use ```tab``` to indent this line.
2. Leave an empty line before ```return 0;```.

File: whoami.c [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #include <stdio.h>
-
-int main() {
```
Comment (jrjang2) [inline whoami.c]: Do not modify the original coding style.

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by TWBestATM** [comment]
I Finish it
.

**Comment by TWBestATM** [comment]
When can I finish it

**Comment by jrjang2** [comment]
Sorry, I miss this commit.

There is a conflict now. Please solve this issue before doing pull request. Update this pull request instead of closing this. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by jrjang2** [comment]
Great!

**Comment by TWBestATM** [comment]
Thank you  very much

**Pull request merged**